Add function to add comment on PR

punkymaniac opened the pull request.

punkymaniac committed:
Add function to add comment on PR

gbonnefille commented:
Thanks for this contribution.

Can you write a minor introduction in the README?

Concerning the naming convention, I started by adding the name `tuleap` in the name of all functions as it seems to be a global scope. Perhaps can we rename this function as `commentTuleapPR`?

punkymaniac committed:
Rename files commentPR to commentTuleapPR

punkymaniac committed:
Add commentTuleapPR description to README

punkymaniac committed:
Change invalid script name on test script

The script commentPR.groovy has been renomed as commentTuleapPR.groovy.
The change has been forgotten in the test script.

punkymaniac committed:
Fix example comment syntax

gbonnefille committed:
Fix function name in example

gbonnefille committed:
Merge branch 'develop' into comment-PR
